code/modules/admin/topic.dm

```diff
@@ -2180,7 +2180,6 @@
 			ptypes += "Crew Traitor"
 			ptypes += "Floor Cluwne"
 			ptypes += "Shamebrero"
-			ptypes += "Dust"
```

Comment:
I think you might've forgot to add this back to line 2168, cause right now it's not able to be picked on anything at all.
https://github.com/ParadiseSS13/Paradise/blob/29acd36a95db76952fc78554403512727d7dc9ed/code/modules/admin/topic.dm#L2168

Reply:
Whoops